.github/workflows/audit.yml

```diff
@@ -0,0 +1,22 @@
+name: Audit
+
+on:
+  pull_request:
+    paths:
+      - package.json
+  push:
+    branches: [master]
+  schedule:
+    - cron: 0 4 * * *
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: ./.github/actions/node/active-lts
+      - run: yarn audit --groups dependencies
```

Comment:
Should we remove the --groups flag?

Reply:
I don't know, I just kept what was already there, and the change to pick only dependencies seemed to be popular when it was discussed this morning.

Reply:
But feel free to leave it - just wanted to make sure you had the context